core/src/defaultconfigs/bareos-sd.d/autochanger/filechanger.conf

```diff
@@ -0,0 +1,6 @@
+Autochanger {
+  Name = "FileStorage"
```

Comment:
2 remarks: we shouldn't call it FileStorage as the name was used previously to describe the device and dir storage.
FileStorageChanger or FileChanger
Then if we keep the CamelCase format for the Name, then the filename should be using the same format

I understand that keeping FileStorage here allow the director config to be kept as it is. But what will append to people having actually a working installation. Moving suddently to a mulitple devices will completely change the behavior, isn't it?

Can we test it in a systemtest?